tests/epyccel/test_epyccel_pyc_math.py

```diff
@@ -0,0 +1,132 @@
+# pylint: disable=missing-function-docstring, missing-module-docstring/
+# pylint: disable=wildcard-import
+import sys
+from numpy.random import randint, uniform
+from numpy import isclose
+from conftest       import *
```

Comment:
This import shouldn't be needed

Reply:
which one?

Reply:
The conftest one. Pytest looks for it automatically

Reply:
ok, I'll remove it.